dht/sharder.hh

```diff
@@ -28,28 +28,58 @@
 
 namespace dht {
 
+
+// Utilities for sharding ring partition_range:s
+
+// A ring_position range's data is divided into sub-ranges, where each sub-range's data
+// is owned by a single range. Note that multiple non-overlapping sub-ranges may map to a
+// single range, and some shards may not receive any sub-range.
```

Comment:
'single range'? Did you mean 'single shard'?

Reply:
yes

Reply:
This is still 'single range' instead of 'single shard'

Reply:
What about this @avikivity

Reply:
Sorry, I thought I fixed it, I'll send a follow up.

Reply:
Ah, it wasn't merged. Fixing and pushing.